.github/workflows/build-wavefront-floconsole-develop.yaml

```
@@ -81,8 +85,25 @@ jobs:
           docker tag rootflo:${{ env.IMAGE_TAG }} ${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ env.IMAGE_TAG }}
           docker push ${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ env.IMAGE_TAG }}
 
+        # Configure Azure credentials and push to ACR
+      - name: Login to Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          client-secret: ${{ secrets.AZURE_CLIENT_SECRET }}
```

actionlint (1.7.11)

[error] input "client-secret" is not defined in action "azure/login@v2". available inputs are "allow-no-subscriptions", "audience", "auth-type", "client-id", "creds", "enable-AzPSSession", "environment", "subscription-id", "tenant-id"

(action)